packages/@react-spectrum/textfield/test/TextArea.test.js

```diff
@@ -47,11 +47,31 @@ describe('TextArea', () => {
     ${'v3 TextArea'}    | ${TextArea}      | ${{isQuiet: true, onChange}}
     ${'v2 TextArea'}    | ${V2TextArea}    | ${{quiet: true, onChange}}
   `('$Name quiet variant automatically adjusts its vertical height on change', ({Component, props}) => {
-    let tree = renderComponent(Component, props);
-    let input = tree.getByTestId(testId);
+    act(() => {
+      renderComponent(Component, props);
```

Comment:
is the surrounding act because of the `onCheckHeight`? curious if this need would go away with the suggestion i made